pelicanconf.py

```diff
@@ -15,8 +15,22 @@
         {'path': 'favicon.ico'},
 }
 
-ARTICLE_URL = '{date:%Y}/{date:%m}/{slug}.html'
-ARTICLE_SAVE_AS = '{date:%Y}/{date:%m}/{slug}.html'
+ARTICLE_URL = '{date:%Y}/{date:%m}/{slug}/'
+ARTICLE_SAVE_AS = '{date:%Y}/{date:%m}/{slug}/index.html'
```

Comment:
**How this change works**

If you navigate to a directory served by a webserver, usually it will just give you a directory listing. So `ARTICLE_URL` as we can see is a directory named, for e.g. an article written in Aug. 2012 called "flooblegarb", `2012/08/flooblegarb/`.

However, if you put an `index.html` in that directory, that file will get served instead. This accomplishes the neat trick in which one can link to a directory (no need for a filename) but get a file served (the index).